### ISSUE TITLE
Plugin doesn't support React 18

### ISSUE THREAD

**Issue by mi5ha** [lbaldy/flipper-plugin-async-storage-advanced#6]

**Comment by jasonohdz1501** [comment]
Yep, apparently is not compatible with 18 react version. I'm using 18.1.0

**Comment by lbaldy** [comment]
Thanks for pointing it out. I am pretty loaded right now - I may get some time towards the end of the month to fix it. In the meantime - PRs welcome.

As a workaround, did you try to install the dependencies using: `--legacy-peer-deps`

**Comment by gfrancischini** [comment]
I'm using it with React 18 and it works. I am using `yarn`. Additionally, it gives me a warning when `yarn install` due to a peer dep `rn-flipper-async-storage-advanced@1.0.4" having incorrect peer dependency "react@^17.0.2"`.

But it works in runtime.

```
"react": "18.0.0",
"react-native": "0.69.7",
```

Would be good to have the package peer dep update
